workers/unity/Packages/com.improbable.gdk.core/Systems/SpatialOSReceiveSystem.cs

```diff
@@ -76,7 +76,15 @@ protected override void OnUpdate()
             {
                 using (var opList = worker.Connection.GetOpList(0))
                 {
-                    Dispatcher.Process(opList);
+                    try
+                    {
+                        Dispatcher.Process(opList);
+                    }
+                    catch (Exception e)
```

Comment:
I think client error was to deal with errors where a callback is given to the worker sdk directly. So it would be for things like deserialisation or connection flow. However for now we can do whatever we want with this.

Although as said above it doesn't matter too much.